tests/conftest.py

```diff
@@ -153,6 +109,28 @@ def _xfail(*args, **kwargs):
     return _xfail
 
 
+@pytest.fixture(scope="session", autouse=True)
+def evm_version(pytestconfig):
+    version = pytestconfig.getoption("evm_version")
+    # note: configure the evm version that we emit code for.
+    # The env will read this fixture and apply the evm version there.
+    evm_opcodes.DEFAULT_EVM_VERSION = version
+    # This should get overridden by anchor_evm_version, but set it anyway
+    evm_opcodes.active_evm_version = evm_opcodes.EVM_VERSIONS[version]
```

Comment:
i don't think we need to do this anymore (as of e4f1c2479ee8a82777aecb929945f641750b1536)